pkg/reconciler/common/releases.go

```diff
@@ -132,9 +136,49 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	manifests := instance.GetSpec().GetManifests()
+	// Create the comma-separated string as the URL to retrieve the manifest
+	if len(manifests) == 0 {
+		return ""
+	}
```

Comment:
Do we need this early-exit code anymore?

Reply:
we can remove it